test/get-headings.test.js

```diff
@@ -0,0 +1,33 @@
+const test = require('tape');
+const getHeadings = require('../lib/get-headings');
+
+test('should get the id from the heading', function(t) {
+    const body = getBody();
+    const headings = getHeadings(body);
+
+    t.equal(headings[0].id, 'hello');
+    t.equal(headings[1].id, 'another');
+    t.end();
+});
+
+test('should get the level from the heading', function(t) {
+    const body = getBody();
+    const headings = getHeadings(body);
+
+    t.equal(headings[0].level,  2);
+    t.equal(headings[1].level,  3);
+    t.end();
+});
+
+test('should get the html from the heading', function(t) {
```

Comment:
it's HTML, so maybe nest some tag inside the heading to check if that works `<h2><em>Hello</em> world</h2>`.